src/PAModel/Checksum/ChecksumMaker.cs

```diff
@@ -299,6 +299,7 @@ private static void ChecksumJson(Context ctx, IHashMaker hash, JsonElement je)
                             }
                             else
                             {
+                                hash.AppendNull();
```

Comment:
>hash.AppendNull(); [](http://example.com/codeflow?start=32&length=18)

Will this require a new hash C9?

Reply:
Yeah, but actually I think I can fix it without doing that. The bug is with DebugTextHashMaker, if you write an object like 
`{key: null, otherKey: 123}`, the json writer complains that you wrote two keys in a row.